libsyclinterface/helper/source/dpctl_error_handlers.cpp

```diff
@@ -24,7 +24,12 @@
 //===----------------------------------------------------------------------===//
 
 #include "dpctl_error_handlers.h"
+#include "../include/dpctl_service.h"
```

Comment:
```suggestion
#include "dpctl_service.h"
```

Avoid relative paths in include statements.